federatedscope/core/auxiliaries/utils.py

```diff
@@ -335,3 +337,31 @@ def merge_dict(dict1, dict2):
             else:
                 dict1[key].append(value)
     return dict1
+
+
+def download_url(url: str, folder='folder'):
```

Comment:
Please check if there is other versions of the functions `download_url` and `extract`. (currently `mf` uses the torch version)

Reply:
The minimal version `requirements-torch1.10.txt` contains `torchvision`,  so I think the usage of `download_url` from it is ok.